components/email/src/Email.svelte

```diff
@@ -73,7 +76,7 @@
     show_number_of_messages: true,
     show_received_timestamp: true,
     show_star: false,
-    show_thread_actions: true,
+    show_thread_actions: false,
```

Comment:
Is there a reason we're changing this default here @pooja169usp ?

Reply:
@AaronDDM yes, we either display `show_thread_actions` or `show_received_timestamp`. In the html we have, if `show_thread_actions` and else if `show_received_timestamp`. Having this default set to true will always display the thread actions in the mailbox.